vehicle/OVMS.V3/components/vehicle_smarteq/src/vehicle_smarteq.cpp

```diff
@@ -292,6 +307,7 @@ void OvmsVehicleSmartEQ::IncomingFrameCan1(CAN_frame_t* p_frame) {
         StandardMetrics.ms_v_bat_range_full->SetValue((float) (_range_est / StandardMetrics.ms_v_bat_soc->AsFloat(0)) * 100.0); // ToDo
       }
       StandardMetrics.ms_v_bat_range_ideal->SetValue((165 * StandardMetrics.ms_v_bat_soc->AsFloat(0)) / 100.0); // ToDo
+      StandardMetrics.ms_v_inv_power->SetValue(mt_evc_hv_energy->AsFloat());
```

Comment:
Hab ich verschoben, da es hier keinen Sinn ergibt es hier zu setzen da der Wert in eq_can_poll gesetzt wird und nicht in dieser can ID.